.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
     'extends': 'airbnb',
+    'rules': {
+      'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
We've gone with `.js` everywhere at Yelp because of [this reasoning](https://github.com/airbnb/javascript/pull/985#issuecomment-239145468). I personally think allowing two file extensions is confusing, but it isn't that big of a deal either way!